### ISSUE TITLE
Suggestion - Prevent well of suffering from dropping items

### ISSUE THREAD

**Issue by Castone22** [WayofTime/BloodMagic#998]
It might be nice if it were possible to apply a reagent or use the tinkerer to stop well of suffering from dropping items and or xp.  It's really common for players to just set it up and let it litter the ground with stuff while they use another spawner and grinder system for items.

**Comment by VT-14** [comment]
The people who are too lazy to set up a Call of the Zephyr ritual to collect the free mob drops are still going to be too lazy to feed the ritual demon will to remove them completely.  In order for this change to be effective, you would have to make no drops the default.  I see that as both good for lag prevention, and bad for punishing people who are collecting and using those mob drops because of a few irresponsible players.

Whatever direction is taken, it should probably also be applied to the Demon Will Crystal growing ritual (whose name I have yet to memorize).

Do the rituals drop xp?  I just assumed they didn't.  Perhaps a will upgrade could be having rituals drop player kill items, and maybe looting as another one?

**Comment by TehNut** [comment]
I don't believe they drop experience or player specific drops (like Blaze Rods)

**Comment by bookerthegeek** [comment]
I would prefer, from a server standpoint, if you defaulted to no mob drops, and an additional  something to have the mobs drop loot.

This way you could have different types of things you could add,  to change from noisily loot to only xp to player drops .

**Comment by KingEdward9830** [comment]
I remember playing on a server and I forgot to put in something to collect the items, nearly crashed the entire server... I agree with bookerthegeek on this one.

**Comment by TehNut** [comment]
I don't think we'll disable drops by default, but if we add an augment to remove them, I'm sure we can have a config to toggle the how it performs.

**Comment by keraldi** [comment]
Config option exists in newest versions.
